docs: how to add new admin route under existing route (#10652)
medusajs/medusa: www/apps/book/app/learn/fundamentals/admin/ui-routes/page.mdx

- Nested routes in setting pages aren't shown in the sidebar to follow the admin's design conventions.
- The `icon` configuration is ignored for the sidebar item of nested UI route to follow the admin's design conventions.

> ### Route Under Existing Admin Route

Reviewer: ```suggestion
### Route Under Existing Admin Route

```

Reviewer: @eugenepro2 there's also this change please (need to add a new line after the header)

> You can locate a route under an existing route. For example, you can locate a route under the orders route in the sidebar.

Reviewer: ```suggestion
You can add a custom UI route under an existing route. For example, you can add a route under the orders route:
```

```tsx title="src/admin/routes/orders/nested/page.tsx"
import { defineRouteConfig } from "@medusajs/admin-sdk"
import { Container, Heading } from "@medusajs/ui"

const NestedOrdersPage = () => {
  return (
    <Container className="divide-y p-0">
      <div className="flex items-center justify-between px-6 py-4">
        <Heading level="h1">Nested Orders Page</Heading>
      </div>
    </Container>
  )
}

export const config = defineRouteConfig({
  label: "Nested Orders",
  nested: "/orders",
})

export default NestedOrdersPage

> ```

Reviewer: ~~~suggestion
```

The `nested` property passed to `defineRouteConfig` specifies which route this custom route is nested under. This route will now show in the sidebar under the existing "Orders" sidebar item.
~~~

> The `nested` property passed to `defineRouteConfig` specifies which route this custom route is nested under. This route will now show in the sidebar under the existing "Orders" sidebar item.

Reviewer: ```suggestion
The `nested` property passed to `defineRouteConfig` specifies which route this custom route is nested under. This route will now show in the sidebar under the existing "Orders" sidebar item.

```

Need to add a new line after the sentence / before the divider
